back/app/services/participation_context_service.rb

```diff
@@ -91,7 +91,7 @@ def posting_idea_disabled_reason_for_project(project, user)
   def posting_idea_disabled_reason_for_context(context, user)
     if !context
       POSTING_DISABLED_REASONS[:project_inactive]
-    elsif !context.ideation? && !context.native_survey?
+    elsif !Factory.instance.participation_method_for(context).posting_allowed?
       POSTING_DISABLED_REASONS[:not_ideation] # TODO: (native surveys) change reason code?
```

Comment:
Do you think we could consider changing the not_ideation reason to something else for i2?